### PR TITLE
Add functionality that enables serialization of null fields in json

### PR DESCRIPTION
**Pull request by robsonperassoli**
This pull request is related to issue #984 .

### CODE REVIEW

**Comment by Turini** [comment]
that sounds great to me. Could you just add some docs? It could be added here:
http://www.vraptor.org/en/docs/view-and-ajax/#ajax-programmatic-version
To do that, you just need to edit the content on `vraptor-site/content/[pt/en]`

**Comment by robsonperassoli** [comment]
Sure! My mistake. Docs added.

File: vraptor-site/content/en/docs/view-and-ajax.html [hunk 1]
```diff
@@ -281,7 +281,22 @@
 result.use(xml()).from(user).excludeAll().serialize(); // excludes all fields
 ~~~ 
 
-You can also add versioning capability to JSON based results.
+The default behavior of JSON serialization is exclude atributes with `null` values, is possible to change this behavior:
```
Comment (Turini) [inline vraptor-site/content/en/docs/view-and-ajax.html]: i'd just change from:

> with `null` values, is possible to change this behavior

to something like:

> with `null` values, but it's possible to change this behavior by calling `serializeNulls` method.

what do you think?

Comment (robsonperassoli) [inline vraptor-site/content/en/docs/view-and-ajax.html]: Sure, a lot better. I'll change...

**Comment by garcia-jj** [comment]
Amazing.

**Pull request merged**